.eslintrc.js

```diff
@@ -13,7 +13,10 @@ module.exports = {
   ],
   rules: {
     '@typescript-eslint/no-empty-interface': 'off',
-    "sort-imports": "error",
+    '@typescript-eslint/space-before-function-paren': [ 'error', 'never' ],
+    'class-methods-use-this': 'off',
```

Comment:
Maybe not this one, depending on what we do in #3 with that helper method.

Reply:
Even if that one changes there are still cases where this can be an issue. Specifically if something implements AsyncHandler but does not use `this` in the `canHandle` function. The linter rule requires it to be static but the interface requires it to be non-static causing a conflict.

Reply:
Oh, the linter isn't smart enough to detect this?...
Ok in that case :-)